content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -758,6 +767,12 @@ with different SELinux labels:
 A cluster admin can use this information to identify pods affected by the planning change and
 proactively opt-out Pods from the optimization (i.e. set `spec.securityContext.seLinuxChangePolicy: Recursive`).
 
+{{< warning >}}
+We strongly recommend clusters that use SELinux to enable this controller and make sure that
+`selinux_warning_controller_selinux_volume_conflict` metric does not report any conflicts before enabling `SELinuxMount`
```

Comment:
are you sure that this metrics will report anything before `SELinuxMount` is enabled?

Reply:
Yes. That's why the whole new controller is introduced. Just to expose metrics + events for Pods that would break *before* `SELinuxMount` is enabled.

Reply:
Got it. Thanks.